docs/release-notes/8.8.asciidoc

```diff
@@ -12,6 +12,7 @@ To view a detailed summary of new features and enhancements we've documented thi
 ==== Known issues
 
 * Setting the `max_signals` value higher than the {kibana-ref}/alert-action-settings-kb.html#alert-settings[`xpack.alerting.rules.run.alerts.max`] value will lead to rule failure.
+* All field types in the the **Fields** browser display as the `unknown` field type.
```

Comment:
```suggestion
* All field types in the **Fields** browser appear as `unknown`.
```
If you know of any more context such as a work around or anywhere else people should look instead, it might be helpful, but otherwise this LGTM.

Reply:
Unfortunately there's no workaround for this one :( It should be fixed in the next patch though.